data/json/items/ammo/metal_rail.json

```diff
@@ -4,16 +4,17 @@
     "type": "AMMO",
     "name": { "str": "rebar rail" },
     "description": "A short piece of ferromagnetic rebar which has been straightened and sharpened on one end.  Could be used as a projectile by a sufficiently powerful electromagnetic weapon.",
-    "weight": "160 g",
-    "volume": "25ml",
+    "weight": "236 g",
```

Comment:
Just multiply it by the count
```suggestion
    "weight": "2360 g",
```

Reply:
> 
> 
> Just multiply it by the count

 My mistake, when testing it seems that it's the volume that needs to change by a factor of 10, not the weight. The weight changes to match after this.
![volume](https://user-images.githubusercontent.com/60374171/90999641-28770d00-e595-11ea-9c41-fadbfa058344.png)